packages/design-system/src/styles/components/_Dropdown.scss

```diff
@@ -1,37 +1,45 @@
 @use '../layout' as *;
 @use '../mixins';
 
+// Wraps all label and dropdown markup
 .ds-c-dropdown {
   max-width: var(--form__max-width);
   position: relative;
 }
 
+// Dropdown uses a <button> to fake a <select> element
+// This styles a <button> to look like a <select> and positions content within
 .ds-c-dropdown__button.ds-c-field {
   appearance: none;
   background-color: var(--dropdown__background-color);
-  background-image: url('../images/arrow-both.svg');
-  background-position: right $spacer-1 center;
-  background-repeat: no-repeat;
-  background-size: var(--dropdown__icon-size);
-  padding-inline-end: 28px;
+  display: flex;
+  gap: 2ch;
+  justify-content: space-between;
   text-align: start;
 
-  &[multiple] {
-    background-image: none;
-  }
-
   // Removes inner outline on select elements in Firefox.
   &:-moz-focusring {
     color: transparent;
     text-shadow: 0 0 0 #000;
   }
+}
+
+// Dropdown previously used a background image for the caret
+// Caret is now an SVG icon that is rotated based on aria-expanded set on its parent
+// Rotation (and potential animation) couldn't be handled on the background element, but can now that it's its own element
+.ds-c-dropdown__caret {
+  flex-basis: 0 0 auto;
+
+  [aria-expanded='true'] & {
+    transform: rotate(180deg);
+  }
 
-  .ds-c-dropdown--open & {
-    border-end-end-radius: 0;
-    border-end-start-radius: 0;
+  [aria-expanded='false'] & {
+    transform: rotate(0deg);
   }
 }
 
+// Styles for the dropdown menu
 .ds-c-dropdown__menu-container {
   background-color: var(--autocomplete__background-color);
   border: var(--text-input__border-width) solid var(--text-input__border-color);
```

Comment:
Something I noticed: Our error border (`--text-input__border-widthh--error`) is 3px wide and the field border (`--text-input__border-width`) is 2px wide. So when an errored dropdown opens, the menu's border is 2px wide (matching the default field's width). 

Is there a reason our border's are off by 1px? I'd argue there's not enough of a difference between these 2 values to be noticeable by our users so perhaps we should have the same width for all field states.

Applies to:
- [CMS theme](https://github.com/CMSgov/design-system/blob/main/packages/design-system-tokens/src/themes/cmsgov-component.ts#L413-L414)
- [Core theme](https://github.com/CMSgov/design-system/blob/main/packages/design-system-tokens/src/themes/core-component.ts#L413-L414)
- [Hgov theme](https://github.com/CMSgov/design-system/blob/main/packages/design-system-tokens/src/themes/healthcare-component.ts#L423-L424)
- [Mgov theme](https://github.com/CMSgov/design-system/blob/main/packages/design-system-tokens/src/themes/medicare-component.ts#L430-L433) (they also set border-width for disabled fields to 1px?)